rskj-core/src/main/java/co/rsk/peg/BridgeMethods.java

```diff
@@ -577,14 +577,14 @@
             activations -> activations.isActive(RSKIP143),
             false
     ),
-    REGISTER_BTC_TRANSFER(
+    REGISTER_FAST_BRIDGE_BTC_TRANSACTION(
```

Comment:
Why this change is NOT A HARD FORK? AFAIK, this change doesn't rerun previous sent transactions

Reply:
It is a HF. (RSKIP 176)